app/javascript/mastodon/components/logo.tsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import logo from 'mastodon/../images/logo.svg';
 
-export const WordmarkLogo = () => (
+export const WordmarkLogo: React.FC = () => (
   <svg viewBox='0 0 261 66' className='logo logo--wordmark' role='img'>
     <title>Mastodon</title>
     <use xlinkHref='#logo-symbol-wordmark' />
   </svg>
 );
 
-export const SymbolLogo = () => (
+export const SymbolLogo: React.FC = () => (
   <img src={logo} alt='Mastodon' className='logo logo--icon' />
 );
 
```

Comment:
Could you remove the default export, and change all imports of those 2 components to named imports?

This is the new standard for Typescript components, as default exports break Hot Module Reloading?

Thanks!

Reply:
Understood!

I've removed the default export in `logo.tsx`.

Additionally, since `navigation_panel.jsx` was using the default import, I've updated it to use the named import instead!